code/game/objects/structures/dropship_equipment.dm

```diff
@@ -659,7 +659,7 @@
 		playsound(loc, firing_sound, 70, 1)
 	var/obj/structure/ship_ammo/SA = ammo_equipped //necessary because we nullify ammo_equipped when firing big rockets
 	var/ammo_accuracy_range = SA.accuracy_range
-	var/ammo_travelling_time = SA.travelling_time //how long the rockets/bullets take to reach the ground target.
+	var/ammo_travelling_time = SA.travelling_time * (GLOB.current_orbit/3) //how long the rockets/bullets take to reach the ground target.
```

Comment:
That's a hellish of a buff, reducing by 66% the travel time? This is almost death sentence to resting xenos if you are 1 second late to react

Reply:
Keep in mind dropship points are affected by orbit too (that's already in the game) so you're also only able to generate 1/3 of the ammo at orbit 1, if it's too ridiculous I can half the affect of orbit on travel time and AMMO but I think they should both be affected by an equal amount

Reply:
We'll see in tm